clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/cache/client/AbstractCacheClient.java

```diff
@@ -89,6 +92,7 @@ public T get(String key) {
    * @return
    */
   private Collection<CacheData> fetchFromCache(String account, String region) {
+    log.debug("fetching all for account '{}' and region '{}'", account, region);
```

Comment:
not a necessity of this PR, but it would be cool if we could add some more debug logging below for things like the number of items in allData or nameMatches?

Reply:
I wanted to be conservative re: adding more here, since they'll apply to every caching agent (which also have logging of their own), but I agree some more strategic debug statements (here or elsewhere in the caching layer) would be good.